Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -487,7 +487,7 @@ open class ComposerVC: _ViewController,
             Animate {
                 self.composerView.confirmButton.isHidden = false
                 self.composerView.sendButton.isHidden = true
-                self.composerView.recordButton.isHidden = self.composerView.sendButton.isHidden
+                self.composerView.recordButton.isHidden = self.composerView.confirmButton.isHidden
```

Comment:
Was this a bug? Why is this change needed?

Reply:
Was this intended? @ipavlidakis

@polqf  Should we add an entry to the changelog since it is a bug fix?

Reply:
We did not allow editing attachments (and voice messages are one of those) before, so this is part of this task